src/messages.js

```diff
@@ -1,6 +1,7 @@
 export const MESSAGES = {
     browserPathNotSet:       'Unable to run the browser. The browser path or command template is not specified.',
     unableToRunBrowser:      'Unable to run the browser. The file at {path} does not exist or is not executable.',
+    unableToFindBrowser:     'Unable to find the browser. "{path}" is not a browser alias or a path to browser executable.',
```

Comment:
Just stylistic remark: `"{browser}" "{path}" is not a browser alias or a path to browser executable.`